android/java/org/chromium/chrome/browser/crypto_wallet/adapters/WalletCoinAdapter.java

```diff
@@ -135,7 +135,8 @@ public void onBindViewHolder(@NonNull WalletCoinAdapter.ViewHolder holder, int p
             } else {
                 onWalletListItemClick.onAccountClick(walletListItemModel);
                 if (mType == AdapterType.SELECT_ACCOUNTS_LIST) {
-                    updateSelectedNetwork(holder.getLayoutPosition());
+                    holder.assetCheck.post(
+                            () -> { updateSelectedNetwork(holder.getLayoutPosition()); });
```

Comment:
This may improve the performance a bit but the issue still can be reproduced because the flow is still same i.e. hiding an icon process is separate from showing the another one, it is visible for the first time in the posted video: https://bravesoftware.slack.com/archives/C02SC5AKTAS/p1655379158901509?thread_ts=1653566500.573419&cid=C02SC5AKTAS

Reply:
It's not reproducible, however I'll try more with more accounts.

Reply:
Tried this on an old device (Xiaomi A1). Will try on more devices
https://user-images.githubusercontent.com/9821527/174680603-e2c622f6-3c23-4020-ae66-150142d9bf57.mp4